packages/lab/stories/badge/badge.doc.stories.mdx

```diff
@@ -70,3 +70,24 @@ import { Badge } from "@salt-ds/lab";
 ## Characteristics Used
 
 <CharacteristicUsage of={Badge} />
+
+# Content
+
+If a numeric annotation exceeds the max value passed as a prop, a plus (+) sign will be displayed after the max value stated. In the case that no max value is passed to the component, the value will truncate after 3 digits and therefore display 999+ for any values over 999.  
+
+If an alpha or text annotation in Badge exceeds four characters an ellipsis is used to truncate the annotation after the fourth character. 
```

Comment:
As discussed in the meeting yesterday the string truncation functionality should probably be removed ( probably in BadgeV2), but these docs are accurate for the code that currently exists